geomesa-spark/geomesa-spark-core/src/main/scala/org/locationtech/geomesa/spark/GeoMesaSpark.scala

```diff
@@ -8,9 +8,11 @@
 
 package org.locationtech.geomesa.spark
 
-import java.io.{BufferedWriter, StringWriter}
+import java.io.{BufferedWriter, OutputStream, StringWriter}
 import java.util.ServiceLoader
 
+import com.vividsolutions.jts.geom.Geometry
+import net.razorvine.pickle.{IObjectPickler, Pickler}
```

Comment:
Was this already on the list of dependencies?

Reply:
it's transitive through spark